src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,525 @@
+        /// or when increasing volume of the target device fails
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void IncreaseVolume()
+        {
+            if (_isConnected == true)
+            {
+                BluetoothAvrcpControlImpl.Instance.IncreaseVolume();
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.RemoteDeviceNotConnected);
+            }
+        }
+
+        /// <summary>
+        /// Decreases volume of target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected
+        /// </remarks>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when decreasing volume of the target device fails
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void DecreaseVolume()
+        {
+            if (_isConnected == true)
+            {
+                BluetoothAvrcpControlImpl.Instance.DecreaseVolume();
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.RemoteDeviceNotConnected);
+            }
+        }
+
+        /// <summary>
+        /// Sends delay report to the target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected
+        /// </remarks>
+        /// <param name="delay">Delay to be sent to target</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when sending delay to the target device fails
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SendDelayReport(uint delay)
+        {
+            if (_isConnected == true)
+            {
+                BluetoothAvrcpControlImpl.Instance.SendDelayReport(delay);
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.RemoteDeviceNotConnected);
+            }
+        }
+
+        ~BluetoothAvrcpControl()
+        {
+            Dispose(false);
+        }
+
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        private void Dispose(bool disposing)
+        {
+            if (disposed)
+                return;
+
+            if (disposing)
+            {
+                // Free managed objects.
+                ConnectionStateChanged -= OnConnectionChanged;
```

Comment:
```suggestion
                BluetoothAvrcpControlImpl.Instance.ConnectionChanged -= OnConnectionChanged;
```

You need to unset others event handler, 
ex) 
``` c#
BluetoothAvrcpControlImpl.Instance.PositionChanged += (s, e) => PositionChanged?.Invoke(this, e);
BluetoothAvrcpControlImpl.Instance.PlayStateChanged += (s, e) => PlayStateChanged?.Invoke(this, e);
BluetoothAvrcpControlImpl.Instance.TrackInfoChanged += (s, e) => TrackInfoChanged?.Invoke(this, e);
```

Reply:
Thank you!